settings/src/lib.rs

```diff
@@ -152,28 +203,32 @@ pub struct StatsServerSettings {
     pub stats_port: u16,
 }
 
-#[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq, Default)]
-pub struct ExitTunnelSettings {
-    pub lan_nics: Vec<String>,
-}
-
+/// This is the main struct for rita
```

Comment:
it would be really cool to have a properly nested diagram here in markdown, but maybe take too long.